cloud/src/meta-service/keys.cpp

```diff
@@ -115,11 +115,12 @@ static void encode_prefix(const T& t, std::string* key) {
         RecycleIndexKeyInfo, RecyclePartKeyInfo, RecycleRowsetKeyInfo, RecycleTxnKeyInfo, RecycleStageKeyInfo,
         StatsTabletKeyInfo,
         JobTabletKeyInfo, JobRecycleKeyInfo, RLJobProgressKeyInfo,
-        CopyJobKeyInfo, CopyFileKeyInfo>);
+        CopyJobKeyInfo, CopyFileKeyInfo, StorageVaultKeyInfo>);
 
     key->push_back(CLOUD_USER_KEY_SPACE01);
     // Prefixes for key families
-    if        constexpr (std::is_same_v<T, InstanceKeyInfo>) {
+    if        constexpr (std::is_same_v<T, InstanceKeyInfo>
+                      || std::is_same_v<T, StorageVaultKeyInfo>) {
```

Comment:
StorageVaultKeyInfo is not int the family of  InstanceKeyInfo.
it should be 
0x01 "storage_vault" $instance_id $resource_id